types/src/build_version/mod.rs

```diff
@@ -120,3 +107,8 @@ impl std::convert::TryFrom<&NodeType> for Version {
         }
     }
 }
+
+/// Returns the version string, e.g., `0.2.2-unstable+git.21146f40`
```

Comment:
I'd drop the git. Another thing, this `unstable` doesn't sound good. What do you think about having `rc` for the release track and `dev` for non-releases?

Reply:
The `git.` is here because Lotus used it too:
```
lotus$ ./lotus -v          
lotus version 1.17.1-dev+interopnet+git.a843c52e3
```
Of course we don't really need to follow them blindly.
This `unstable` doesn't sound good, yes. We could remove this part entirely. Or I would prefer that we add `+debug` for a debug build. This can be useful to users or us.

For the release naming convention (unstable/alpha/beta/rc etc...) this should be more something we handle with git tags.

Reply:
Will you change `unstable` to `+debug` in this PR?

Reply:
If it's ok to you, then yes. It will be more useful than `unstable`.

Reply:
Yes, please.